exams/pearson.py

```diff
@@ -23,6 +23,7 @@
     delimiter='\t',
 )
 
+PEARSON_DATE_FORMAT = "%Y/%m/%d"
```

Comment:
you seem to have a lot of constants in this module. Consider move them to a different `constants.py` module.